{{cookiecutter.project_name}}/config/inverse_predicate_map.yml

```diff
@@ -0,0 +1,14 @@
+inverse_predicate_map:
```

Comment:
Remind me about the use case of having this in the mapping commons as a separate file? Forgot it :D

Reply:
This file is one of the required inputs for the `sssom invert` function (https://mapping-commons.github.io/sssom-py/cli_usage.html#sssom-invert). Would it be better to add an "invert_mappings" option to the cookiecutter.json file and only include this file (and the yet-to-be-written corresponding invert commands in the Makefile) if the user selects "yes" for the option?

Reply:
I added the template variable `inverse_mappings` to cookiecutter.json with "No" as the default option. Now the inverse_predicate_map.yml file will only be added if `inverse_mappings = Yes`.